src/main/kotlin/org/jitsi/rtp/rtp/RtpHeader.kt

```diff
@@ -72,7 +58,7 @@ class RtpHeader {
             (buf[baseOffset].toInt() and 0xC0) ushr 6
 
         fun setVersion(buf: ByteArray, baseOffset: Int, version: Int) {
-            buf[baseOffset] = (buf[baseOffset].toInt() or ((version shl 6) and 0xC0)).toByte()
+            buf[baseOffset] = ((buf[baseOffset].toInt() and 0x3F) or ((version shl 6) and 0xC0)).toByte()
```

Comment:
i've actually used the normal mask for the field (0xC0 here) and taken the inverse of it when doing this in the rest of the fields (that way we can use one mask for each field--which i wondered if we should have constants for)) so let's do that here to be consistent.